projects/isy-angular-widgets/src/lib/incomplete-date/incomplete-date.component.ts

```diff
@@ -91,6 +91,12 @@ export class IncompleteDateComponent implements ControlValueAccessor, Validator,
    */
   @Input() transferISO8601 = false;
 
+  /**
+   * The date format "00.00.0000" for unknown dates will no longer be supported in the future.
+   * It will be phased out and should no longer be used.
```

Comment:
```suggestion
   * It will be phased out and should no longer be used.
   *
   * @deprecated using the format "00.00.0000" for unknown dates is deprecated and will be removed in the future. Use "xx.xx.xxxx" instead.
```